var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -198,6 +198,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
         variant(dev, default=dflt, description=desc)
     conflicts("+cuda", when="+rocm", msg="CUDA and ROCm are not compatible in Kokkos.")
     depends_on("intel-oneapi-dpl", when="+sycl")
+    depends_on("rocthrust", when="+rocm")
```

Comment:
Shouldn't this be
```suggestion
depends_on("rocthrust", when="@4.3: +rocm")
```
?

Reply:
I can't speak to the Spack syntax but you are correct this dependency is only from Kokkos 4.3 release

Reply:
You are true, the requirement is only from version 4.3.